forge-gui/res/cardsfolder/upcoming/smelting_vat.txt

```diff
@@ -0,0 +1,10 @@
+Name:Smelting Vat
+ManaCost:4
+Types:Artifact
+A:AB$ Dig | Cost$ 1 T Sac<1/Artifact.Other/another artifact> | DigNum$ 8 | Reveal$ True | RememberRevealed$ True | DestinationZone$ Library | NoMove$ True | SubAbility$ DBChangeZone 
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ Remembered | Origin$ Library | Destination$ Battlefield | SubAbility$ DBCleanup | MaxTotalTargetCMC X | ChangeNum$ 2 | ChangeType$ Artifact.nonCreature+YouOwn | Optional$ True | RestRandomOrder$ True | DestinationZone2$ Library | LibraryPosition$ -1
```

Comment:
ChangeZone doesn't have `RestRandomOrder`

please try to avoid mixing effects and params too freely, it's really hard to spot